etc/config/kotlin.amazon.properties

```diff
@@ -3,7 +3,7 @@ compilerType=kotlin
 versionFlag=-version
 objdumper=/opt/compiler-explorer/jdk-21.0.0/bin/javap
 instructionSet=java
-defaultCompiler=kotlinc1920
+defaultCompiler=kotlinc2000
```

Comment:
Should the native "k2" thing you added also have a configuration somewhere?

Reply:
The PR enabling native stuff is not ready yet: https://github.com/compiler-explorer/compiler-explorer/pull/6227
I expect it to hop on the recent k2 compiler once the PR will be ready.